docs/plugins/concepts/react.md

```diff
@@ -112,7 +112,7 @@ With this simple patch, we will log out the return value on ever render call but
 ![return_value_expanded](./img/return_value_expanded.png)
 :::
 
-What you see here if a fairly typical result of one of these render calls. Take a second and get familiar with the structure, it's likely you'll be seeing a lot more of them going forward. However, since we want to see where to add our component, expand the tree out like we did above in the second image.
+What you see here if a fairly typical result of one of these render calls. Take a second and get familiar with the structure, its likely you'll be seeing a lot more of them going forward. However, since we want to see where to add our component, expand the tree out like we did above in the second image.
```

Comment:
This should stay `it's`. `it's` is the contraction of `it is` while `its` is a possessive pronoun.